### DISCUSSION TITLE
I2CSCAN command not work

### DISCUSSION THREAD

**Discussion by stoneice** [arendst/Tasmota#17718] [Q&A]
Hello everyone , I am a new one .
I have ESP32 board . I flash the ESP32 and connect the wifi . It works well .
I was learning some command but I found some commands not work , like power , i2csacn . 
It seems i2cscan is not a valid command.
Does ESP32 not support by Tasmota ?
What should I do ?

**Comment by barbudor** [comment]
Learning command is good but Tasmota documentation doesn't start with commands
You have a configuration step to go through first as some command are disabled until the right feature is configured

Did you configure the GPIO to include I2C_SCL/I2C_SDA ?
Did you configure the GPIO to include a Relay ?
